twilio/4.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,43 @@
+= Twilio Connector 4.0 - Mule 4
+:page-aliases: connectors::twilio/twilio-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Twilio is a cloud-based service that enables powerful communication between mobile devices, applications, services, and systems throughout the business to bridge the gap between conventional communication.
+
+Anypoint Connector for Twilio (Twilio Connector) provides an interface to the Twilio REST API, which enables you to create connectivity between Twilio and third party SaaS applications, CRM solutions, and more.
+
+For compatibility information and fixed issues, see the Twilio Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* The Twilio Messaging API
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+Before creating an app, you must have:
+
+* An active Twilio trial or paid account
+* Credentials to access the Twilio target resource
+
+[[create-twilio-account]]
```

Comment:
You could remove this reference if it's not used previously in the same page. Unless, it's used from another page.